examples/step-67/doc/intro.dox

```diff
@@ -0,0 +1,680 @@
+$\mathbf{w}_h^{n}$ at time $t^n$ to the new time $t^{n+1}$ with $\Delta t =
+t^{n+1}-t^n$:
+@f[
+\begin{aligned}
+\mathbf{K}_1 &= \mathcal M^{-1} \mathcal L_h\left(t^n, \mathbf{w}_h^n\right),
+\\
+\mathbf{K}_2 &= \mathcal M^{-1} \mathcal L_h\left(t^n+c_2\Delta t,
+                       \mathbf{w}_h^n + a_{21} \Delta t \mathbf{K}_1\right),
+\\
+&\vdots \\
+\mathbf{K}_s &= \mathcal M^{-1} \mathcal L_h\left(t^n+c_s\Delta t,
+  \mathbf{w}_h^n + \sum_{j=1}^{s-1} a_{sj} \Delta t \mathbf{K}_j),
+\\
+\mathbf{w}_h^{n+1} = \mathbf{w}_h^n + \Delta t\left(b_1 \mathbf{K}_1 +
+b_2 \mathbf{K}_2 + \ldots + b_s \mathbf{K}_s\right).
+\end{aligned}
+@f]
+The vectors $\mathbf{K}_i$, $i=1,\ldots,s$, in an $s$-stage scheme are
+evaluations of the operator at some intermediate state and used to define the
+end-of-step value $\mathbf{w}_h^{n+1}$ via some linear combination. The scalar
+coefficients in this scheme, $c_i$, $a_{ij}$, and $b_j$, are defined such that
+certain conditions are satisfied for higher order schemes, the most basic one
+being $c_i = \sum_{j=1}^{i-1}a_{ij}$. The parameters are typically collected in
+the form of a so-called Butcher tableau as (for a five-stage scheme)
+@f[
+\begin{array}{c|ccccc}
+0 \\
+c_2 & a_{21} \\
+c_3 & a_{31} & a_{32} \\
+c_4 & a_{41} & a_{42} & a_{43} \\
+c_5 & a_{51} & a_{52} & a_{53} & a_{54} \\
+\hline
+& b_1 & b_2 & b_3 & b_4 & b_5
+\end{array}
+@f]
+
+In this tutorial program, we use a subset of explicit Runge--Kutta methods,
+so-called low-storage Runge--Kutta methods (LSRK). The motivation for
+low-storage variants is the fact that as the number of stages grow, there is
+an increasing demand for additional storage due to the intermediate vectors
+$\mathbf{K}_i$ that are needed both for the definition of the next stage
+vector and the final update of $\mathbf{w}_h^{n+1}$. In order to reduce the
+storage requirements, additional structure in the coefficients is assumed. In
+the variant used by reference @cite KennedyCarpenterLewis2000, the assumption
+is to use Butcher tableaus of the form
+@f[
+\begin{array}{c|ccccc}
+0 \\
+c_2 & a_1 \\
+c_3 & b_1 & a_2 \\
+c_4 & b_1 & b_2 & a_3 \\
+c_5 & b_1 & b_2 & b_3 & a_4 \\
+\hline
+& b_1 & b_2 & b_3 & b_4 & b_5
+\end{array}
+@f]
+With such a definition, the update to $\mathbf{w}_h^n$ shares the storage with
+the information for the intermediate values $\mathbf{K}_i$. Starting with
+$\mathbf{w}^{n+1}=\mathbf{w}^n$ and $\mathbf{T}_1 = \mathbf{w}^n$, the update
+in each of the $s$ stages simplifies to
+@f[
+\begin{aligned}
+\mathbf{K}_i &=
+\mathcal M^{-1} \mathcal L_h\left(t^n+c_i\Delta t, \mathbf{T}_{i} \right),\\
+\mathbf{T}_{i+1} &= \mathbf{w}_h^{n+1} + \Delta t a_i \mathbf{K}_i,\\
+\mathbf{w}_h^{n+1} &= \mathbf{w}_h^{n+1} + \Delta t b_i \mathbf{K}_i.
+\end{aligned}
+@f]
+Besides the vector $\mathbf w_h$ that is successively updated, this scheme
+only needs two auxiliary vectors, namely the vector $\mathbf{K}_i$ to hold the
```

Comment:
By the way, if you were renaming cells to use the symbol `$K$`, you might want to also use `\mathbf{k}` with a lower-case `k` for the RK stages. That happens also to be the notation used at the wikipedia site.

Reply:
Changed to lower-case `\mathbf{k}`.